src/v/raft/consensus.h

```diff
@@ -286,9 +286,13 @@ class consensus {
 
     storage::log& log() { return _log; }
 
-    bool are_heartbeats_suppressed(vnode) const;
-
-    void suppress_heartbeats(vnode, follower_req_seq, bool);
+    /**
```

Comment:
awesome! 

Can you break this up into 2 commits.

1. remove methods
2. add comments

Reply:
does the code compile? looks like you just removed the methods.

Reply:
This is merge skew. Will fix